src/model/log_prob.jl

```diff
@@ -206,13 +200,15 @@ function state_log_likelihood(is_star::Bool,
                            gal_mcs_vec[pixel.n], star_mcs_vec[pixel.n])
 
         # compute the background rate for this pixel
-        background_rate = tile.epsilon_mat[pixel.h, pixel.w]
+        background_rate    = tile.epsilon_mat[pixel.h, pixel.w]
         background_sources = tile_sources[tile_sources.!=1]
         for s in background_sources
-            flux_s = variational_params_to_fluxes(s, vp)
-            rate_s = is_star ? model_vars.fs0m_vec[s].v : model_vars.fs1m_vec[s].v
-            rate_s *= flux_s[pixel_band]
-            background_rate += rate_s
+            println("background s: ", s)
```

Comment:
We need to avoid print statements, not just for clutter...they actually cause Julia to crash when it has multiple threads. I wonder when they'll have a thread-safe println statement.